python/ml4ir/applications/classification/pipeline.py

```diff
@@ -92,9 +92,10 @@ def get_relevance_dataset(self, parse_tfrecord=True, preprocessing_keys_to_fns={
 
         NOTE: Override this method to create custom dataset objects
         """
-        # Adding one_hot_vectorizer needed for classification.
+        # Adding one_hot_vectorizer needed for classification
         preprocessing_keys_to_fns = {
-            "one_hot_vectorize_label": get_one_hot_label_vectorizer(self.feature_config.get_label(), self.file_io)
+            "one_hot_vectorize_label": get_one_hot_label_vectorizer(self.feature_config.get_label(), self.file_io),
+            "split_and_pad_string": split_and_pad_string
```

Comment:
This was missing from classification hook (although it was used in ep models).

Reply:
You don't have to add predefined functions. ml4ir will pick it up automatically.

Reply:
Great. Why do we have to add one_hot_vectorize_label ? Because the tf.function is nested inside ?

Reply:
@Ullimague should be able to answer this better.